.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,73 @@
+name: End to end tests
+
+on:
```

Comment:
I guess there should be two conditions: pull request on changes in e2e-tests package, and on release (plus holding release if check fails)

Reply:
Removed running tests on pr in ZbayLite but the second part (on release and holding release) will be done in other PRs